uninstall.sh

```diff
@@ -186,20 +186,18 @@ uninstall_from_existing_repositories() {
 }
 
 #####################################################
-# Uninstall from all repositories in
-#   `autoupdate.registered` which gets deleted
-#    at the end.
+# Uninstall from all registered repositories
 #
 # Returns: 0
 #####################################################
 uninstall_from_registered_repositories() {
 
-    LIST="$INSTALL_DIR/autoupdate/registered"
+    LIST="$INSTALL_DIR/registered"
```

Comment:
With this change, we'll leave the previous file alone. Perhaps delete both the old and the new path?

Reply:
Good point